.github/workflows/cicd.yaml

```diff
@@ -0,0 +1,25 @@
+name: FrontEnd CICD
+
+on: [pull_request]
+
+jobs:
+  run-checking:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checking version of Tools
+        run: |
+          node -v
+          npm -v
+          aws --version
+      - name: Get repo
+        uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - name: Install dependencies
+        run: npm install
+      - name: Running typescript compiler, eslint and prettier
+        run: npm run lint
+      - name: Running Jest
+        run: npm run test
```

Comment:
how about do the cache dependencies ? (source: **https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows**)

```
- name: Cache Dependencies
        uses: actions/cache@v2
        with:
          path: ~/.npm
          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
          restore-keys: |
            ${{ runner.os }}-node-
```